TOPIC 1/shopping/Offer.java

```diff
@@ -0,0 +1,75 @@
+package shopping;
+
+import discount.DiscountHandler;
+import payment.PaymentMethod;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Offer {
```

Comment:
Fijate lo que decia la consigna: offer that includes other individual items or other offers. Aqui deberias aplicar otro patron.